codebase/superdarn/src.bin/tk/tool/make_lmfit.1.0/make_lmfit.c

```diff
@@ -163,6 +163,11 @@ int main(int argc,char *argv[])
 
   if (old) {
       rawfp=OldRawOpen(argv[arg],NULL);
+      /* Error code for num_bytes less than 0 */
+      if (rawfp->rawread==-2) {
```

Comment:
I think I found my problem. What if rawfp returns NULL? This condition with segfault.
Possible solution 
//TODO:  using error codes will make this simpler
if (rawfp != NULL)
    if (rawfp->rawread == -2) 
           free(rawfp);
           ......
else
    fprintf(stderr,"file not found\n");
    .....

Reply:
Fixed between f978ced and e436a98.